pyproject.toml

```diff
@@ -1,5 +1,40 @@
-[tool.bandit]
-skips = ["B101", "B110", "B311"]
-
 [tool.black]
```

Comment:
We don't use black anymore, so remove it

Reply:
I left the black configuration in case some users want to use it for formatting. As ruff is compatible with black I don't see a problem with allowing that.